### PR TITLE
Input Component: Radio Button and checkbox

### PR DESCRIPTION
**Pull request by varun13169**

### CODE REVIEW

File: library/input/input.css [hunk 1]
```diff
@@ -65,3 +65,150 @@
 
 
 /* Search */
+
+
+/* Checkbox Starts*/
+/* https://www.w3schools.com/howto/howto_css_custom_checkbox.asp */
```
Comment (zoef-yt) [inline library/input/input.css:71]: This line should be removed

**Review by zoef-yt — APPROVED** [review]

**Review by humzahasan — COMMENTED** [review]
Extra comments can be removed while refactoring. Code looks good, unable to see preview thou.

**Review by humzahasan — APPROVED** [review]

File: components/input/input.html [hunk 2]
```diff
@@ -134,5 +134,59 @@ <h3 class="comp-lib-h2">Input Search box</h3>
         
         </div>
     </div>
+
+    <h3 class="comp-lib-h2">Check Box</h3>
+    <div class="component_demo_container">
+    <div>
+        <!-- Checkbox Component Starts -->
+        <label class="dui-inp-chkbox dui-util-txt-sm">Checkbox 101
```
Comment (itsmebipul00) [inline components/input/input.html:142]: You could use for attributes in all label matching the id attribute in input for accessibility

**Comment by itsmebipul00** [comment]
Great work! Good to go. Small things: you could use rems in more places and class names that makes sense to a new person reviewing your code.

File: library/input/input.css (hunk 1 above)
Comment (varun13169) [reply library/input/input.css:71]: Nope, this is for reference in the future as well.

File: components/input/input.html (hunk 2 above)
Comment (varun13169) [reply components/input/input.html:142]: I left it out for the user, most probably will be making it part of post MVP.
Thanks.

**Pull request merged**